src/title/titleSelectors.js

```diff
@@ -14,14 +14,14 @@ export const DEFAULT_TITLE_BACKGROUND_COLOR = 'transparent';
  * If `narrow` is a stream or topic narrow, this is based on the stream color.
  * Otherwise, it takes a default value.
  */
-export const getTitleBackgroundColor = (narrow?: Narrow): Selector<string> =>
-  createSelector(
-    getSubscriptions,
-    subscriptions => {
-      if (!narrow || !isStreamOrTopicNarrow(narrow)) {
-        return DEFAULT_TITLE_BACKGROUND_COLOR;
-      }
-      const streamName = narrow[0].operand;
-      return (subscriptions.find(sub => streamName === sub.name) || NULL_SUBSCRIPTION).color;
-    },
-  );
+export const getTitleBackgroundColor: Selector<string, Narrow | void> = createSelector(
+  (state, narrow) => narrow,
+  state => getSubscriptions(state),
+  (narrow, subscriptions) => {
+    if (!narrow || !isStreamOrTopicNarrow(narrow)) {
+      return DEFAULT_TITLE_BACKGROUND_COLOR;
+    }
+    const streamName = narrow[0].operand;
+    return (subscriptions.find(sub => streamName === sub.name) || NULL_SUBSCRIPTION).color;
+  },
+);
```

Comment:
See the first couple of commits in #3914 :smile: 

Actually, in particular https://github.com/gnprice/zulip-mobile/commit/954e6b72b8e430dfa880b755d9c42b60f7b52140 is a good example to look at for the comments I made above about a couple of selectors where the caching isn't useful.